tests/src/core/particle/RateOverDistance.test.ts

```
@@ -0,0 +1,169 @@
+import {
+  Camera,
+  Engine,
+  Entity,
+  ParticleMaterial,
+  ParticleRenderer,
+  ParticleStopMode
+} from "@galacean/engine-core";
```

ESLint

[error] Replace `⏎··Camera,⏎··Engine,⏎··Entity,⏎··ParticleMaterial,⏎··ParticleRenderer,⏎··ParticleStopMode⏎` with `·Camera,·Engine,·Entity,·ParticleMaterial,·ParticleRenderer,·ParticleStopMode·`

(prettier/prettier)